ci/generate_changelog.py

```diff
@@ -0,0 +1,209 @@
+"""OS module for file operations,
+argparse module for command line arguments parsing,
+sys module for graceful exiting,
+regex for searching for old series name. re is does not support unicode characters,
+Optional for optional type."""
+
+import os
+import sys
+import argparse
+from typing import Optional
+import regex
+
+
+# The version of this script just in case the user needs to know.
+SCRIPT_VERSION = "v1.0.0"
```

Comment:
The idea was to keep it just in case something is wrong with the script and someone wants to make sure the version is correct, but since this is an interpreted language, I suppose you can just cat the script for that, so I'll remove it.